### ISSUE TITLE
[Usability] A11y_Manage.IIS.Net_Local IIS-Server: Visual appearance of links inside the grid is not proper.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#185]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Local IIS" page tab and Press Enter.
5. Verify that appearance of all the controls are as per UX design principles.

**Actual Result:**
Visual appearance of links inside the grid is not proper. Links visually appear like a static data inside the grid.

**Expected Result:**
To clearly indicate links, underline them when possible. There should be some visual state change other than color when hovering or focusing on links. Do not rely on color or color changes alone to indicate links.

**Note**: This Kind of scenario is present throughout the application.

**User Impact:**
User may assume the links to be just a static text provided for information purpose. They will not try to select the links and will not be able to go to the destination of the link.

**Attachment:**
![bug_185_usability issue regarding links](https://user-images.githubusercontent.com/38521133/44843258-84884e80-ac65-11e8-9566-a01cb0091450.png)

**Comment by jhkimnew** [comment]
Fixed
